H2O-3/introduction-to-machine-learning-with-h2o-3-automl/introduction-to-machine-learning-with-h2o-3-automl.md

```diff
@@ -1,576 +1,938 @@
+
+After exploring the results for our classification problem, let’s use AutoML to explore a regression use-case.
+
+**Task 4 - R Version**
+
+We already have our train and test sets, so we just need to choose our response variable, as well as the predictors. We will do the same thing that we did for the first tutorial.
+
+~~~r
+# Identify predictors and response for the classification use case
+ignore <- c("DELINQUENT", "PREPAID", "PREPAYMENT_PENALTY_MORTGAGE_FLAG", "PRODUCT_TYPE")
+y <- "DELINQUENT"
+
+x <- setdiff(colnames(train), ignore)
+~~~
+
+Now we are ready to run AutoML. Below you can see some of the default parameters that we could change for AutoML
+
+~~~markdown
+H2OAutoML(nfolds=5, max_runtime_secs=3600, max_models=None, stopping_metric='AUTO', stopping_tolerance=None, stopping_rounds=3, seed=None, project_name=None)
+~~~
+
+As you can see, H2O AutoML is designed to have as few parameters as possible, which makes it very easy to use. For this experiment, we will set values for the maximum runtime, and we will choose fifteen minutes, the seed, and the project name; Also, we will set `max_models = 25` so that AutoML runs for fifteen minutes or only trains twenty-five models. Lastly, we will set `sort_metric` to `AUC` so that the leaderboard gets sorted by AUC.
+
+~~~r
+# Run AutoML for 25 base models or 15 minutes
+aml_cl <- h2o.automl(x = x,
+                     y = y,
+                     training_frame = train,
+                     max_runtime_secs = 900,
+                     max_models = 25,
+                     seed = 42,
+                     project_name = "classification2",
+                     sort_metric = "AUC")
+~~~
+
+The only required parameters for H2O's AutoML are, `y` `training_frame,` and `max_runtime_secs,` which let us train AutoML for ‘x’ amount of seconds and/or `max_models,` which would train a maximum number of models. Please note that  `max_runtime_secs` has a default value, while `max_models` does not. The seed is the usual parameter that we set for reproducibility purposes. We also need a project name because we will do both classification and regression with AutoML. Lastly, we are using the default number of folds for cross-validation.
+
+The second line of code has the parameters that we need in order to train our model. For now, we will just pass x, y, and the training frame. Please note that the parameter `x` is optional because if you were using all the columns in your dataset, you would not need to declare this parameter. The `leaderboard frame` can be used to score and rank models on the leaderboard, but we will use the validation scores to do so because we will check the performance of our models with the test set.
+
+Below is a list of optional parameters that the user could be set for H2O’s AutoML
```

Comment:
rewrite this: "Below is a list of optional parameters that the user could be set for H2O’s AutoML"

to this: "Below is a list of optional parameters that the user could set for H2O’s AutoML"